### PR TITLE
Filename updated

### PR DESCRIPTION
**Pull request by SwatiRaoDataEnthusiast**
I have updated my filename to the required format.Please accept PR and merge

### CODE REVIEW

File: docs/TEAM.md [hunk 1]
```diff
@@ -13,7 +13,9 @@
 |[Manohar Rajaram](https://github.com/manohar52) |I am from India but currently pursuing my Masters in computer science in US. I am quite new to Javascript. I have some experience in HTML, CSS and PHP.|
 |[Sheetal Pandey](https://github.com/Sheetal777) |A keen learner from India pursuing engineering. Good with Python but new to Web Developement. |
 |[Shekhar Ranjan](https://github.com/shekhar10feb)|I am from India, have basic knowledge of HTML, CSS, Java, SQL, Electronics Engineer, currently learning JavaScript, a beginner in Git and GitHub, like to draw, paint and write stories.|
+|[Swati Rao](https://github.com/SwatiRaoDataEnthusiast)|I am from India,I am familiar with HTML,CSS and can learn Node.JS, javascript etc. This is the my first open source project TWINDLE. Really excited!!|
 |[Nahuel Alfano](https://github.com/NahuelAlfano)|I am from Argentina, beginner at github and open source projects. I have basic knowledge of HTML, CSS, Javascript, C#, C++, currently learning Python. |
+>>>>>>> upstream/main
```
Comment (pranavgoel29) [inline docs/TEAM.md:18]: remove this line

**Comment by Rakesh-4** [comment]
Hey swati, can you please remove the line 18 and push ur new changes and it should be good to go next time. Thanks

File: docs/TEAM.md (hunk 1 above)
Comment (SwatiRaoDataEnthusiast) [reply docs/TEAM.md:18]: okay

**Comment by SwatiRaoDataEnthusiast** [comment]
I have removed line 18

**Review by pranavgoel29 — APPROVED** [review]

**Comment by pranavgoel29** [comment]
@sarathkumar6 , @johnjacobkenny and @proful can anyone of you merge it.

File: docs/TEAM.md (hunk 1 above)
Comment (SwatiRaoDataEnthusiast) [reply docs/TEAM.md]: @Rakesh-4 I have removed line 18 as you asked. Please review and approve for merging

**Comment by SwatiRaoDataEnthusiast** [comment]
> @sarathkumar6 , @johnjacobkenny and @proful can anyone of you merge it.

Please do this. Thanks!

**Review by EYRAM565 — APPROVED** [review]

**Review by pranavgoel29 — CHANGES_REQUESTED** [review]
Please pull changes once again and push them
as some new PR have been merged after this

**Comment by SwatiRaoDataEnthusiast** [comment]
I have resolved by updating the team.md file please approve and merge

**Review by pranavgoel29 — APPROVED** [review]

**Comment by SwatiRaoDataEnthusiast** [comment]
@johnjacobkenny @proful  @sarathkumar6 please approve and merge. Thanks!

**Review by Akshay2996 — APPROVED** [review]
It is ready to be merged! All the changes have been successfully done

**Comment by johnjacobkenny** [comment]
Please fix conflicts

**Comment by SwatiRaoDataEnthusiast** [comment]
Resolved conflicts now .Please merge

**Comment by SwatiRaoDataEnthusiast** [comment]
@johnjacobkenny

**Pull request merged**